### ISSUE TITLE
Using the editors.knockout-extensions, biding on a null value on certain control don't work.

### ISSUE THREAD

**Issue by seawerst** [IgniteUI/ignite-ui#1076]
### Description  
Using the editors.knockout-extensions, binding on a null value on certain control don't work.

### Steps to reproduce  

On the Update method of the knockout bindingHandlers for the igNumericEditor, igCurrencyEditor and igPercentEditor, a test is made for an empty string "if (value !== "") {" but it should also allow a null value as it's tested in the igDateEditor and igDatePicker.

**Comment by Lipata** [comment]
Hi, Marc.

We will work on fixing that. Thank you!

**Comment by Lipata** [comment]
The fix will be applied to the 17.1 and 17.2 versions.

**Comment by Lipata** [comment]
To reproduce the problem, follow the steps.

ViewModel:

```js
this.nullValue = ko.observable(null);
```

View:

```html
<input id="nullBoundEditor" data-bind="igNumericEditor: { value: nullValue, allowNullValue: true }" />
```

Execute:

```js
model.nullValue(23);
model.nullValue(null);
```

Expected Result:

```js
$editor.igNumericEditor("value") === null
```

This should be valid for igNumericEditor, igPercentEditor and igCurrencyEditor.
